.github/workflows/update-tor.yml

```diff
@@ -0,0 +1,43 @@
+name: Tor package updates
+on:
+  schedule:
+    - cron: "0 6 * * *"
+jobs:
+    check-for-tor-updates:
+        runs-on: ubuntu-latest
+        container: debian:bullseye-backports
```

Comment:
I copied this from the CircleCI job, but I did not research whether the `backports` image specifically is still needed, or if we should switch to a different one.

Reply:
Done & confirmed that `gh` is still successfully authenticated.